### PR TITLE
Add codeowners for Video Indexer

### PR DESCRIPTION
**Pull request by dazilb**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -260,6 +260,9 @@
 # PRLabel: %Web Apps
 /specification/web/ @naveedaz @Azure/azure-app-service-control-plane
 
+# PRLabel: %Video Indexer
+/specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
```
Comment (weshaggard) [inline .github/CODEOWNERS:264]:
```
Error(s) on line 264
Source Line: /specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
  -dazilb is not a public member of Azure.
  -tshoham is not a public member of Azure.
  -omerhaimov is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
  -bermanraz is not a public member of Azure.
  -Einav94 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.

There were linter errors. Please visit https://aka.ms/azsdk/codeownersLinterErrors for guidance on how to handle them.
```

Comment (dazilb) [reply .github/CODEOWNERS:264]: dazilb is a public member 
image

is this the correct group?

Comment (weshaggard) [reply .github/CODEOWNERS:264]:
```
Error(s) on line 263
Source Line: # PRLabel: %Video Indexer
  -'Video Indexer' is not a valid label for this repository.

Error(s) on line 264
Source Line: /specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
  -tshoham is not a public member of Azure.
  -omerhaimov is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
  -Einav94 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.

There were linter errors. Please visit https://aka.ms/azsdk/codeownersLinterErrors for guidance on how to handle them.
```

@dazlib looks like your account is corrected but there are still other issues.

Comment (dazilb) [reply .github/CODEOWNERS:264]: my account (and everyone else''s) is public, so it might be a problem with the linter

Comment (dazilb) [reply .github/CODEOWNERS:264]: @weshaggard thanks! q: why is Video Indexer an invalid label?

Comment (weshaggard) [reply .github/CODEOWNERS:264]: That label doesn't exist in the repo https://github.com/Azure/azure-rest-api-specs/labels?q=Video+Indexer

Comment (dazilb) [reply .github/CODEOWNERS:264]: I see, thanks so much :)

Comment (dazilb) [reply .github/CODEOWNERS:264]: I added it but it's still failing. How long does it take to propagate?

Comment (dazilb) [reply .github/CODEOWNERS:264]: @weshaggard all checks have passed, can you please approve? :)

**Review by weshaggard — APPROVED** [review]

**Pull request merged**